frontend/model/contracts/group.js

```diff
@@ -413,6 +413,9 @@ sbp('chelonia/defineContract', {
     groupShouldPropose (state, getters) {
       return getters.groupMembersCount >= 3
     },
+    groupDistributionStarted (state, getters) {
+      return Object.keys(getters.groupPeriodPayments).length >= 2
```

Comment:
I think `new Date().toISOString() >= getters.groupSettings?.distributionDate` might be more reliable in some edge cases where we've just entered the new period but the change hasn't been registered yet

Reply:
@snowteamer `new Date()` is never allowed inside of contract getters. They must be deterministic, same for `process` functions.

Reply:
We can make it this way, I think.
```
groupDistributionStarted (state, getters) {
  return (cDateInISO: string) => cDateInISO >= getters.groupSettings?.distributionDate
}
```

Reply:
@taoeffect You are right. Then maybe we can use Silver-IT's idea, or find a way so that his previous suggestion (`Object.keys(getters.groupPeriodPayments).length >= 2`) will always be correct